layout/templates/wu_story.mustache

```diff
@@ -43,7 +43,36 @@
           </picture>
         </figure>
       </div>
-    
+      
+      <div id="national-static-pie-mobile-container">
+        <div id="national-static-pie-mobile-content">
+          <p>
+          <strong>1. Public Supply</strong> water withdrawals, mostly for Domestic use, are highest in counties with large numbers of people.
+          </p>
+          <p>
+            <strong>2. Aquaculture, mining, self-supplied domestic, and livestock</strong> water uses are distributed unevenly across the US. There are large withdrawals for aquaculture along the Snake River in southern Idaho.
+          </p>
+          <p>
+            <strong>3.</strong> Larger withdrawals in Alaska in the “other” categories are for <strong>aquaculture and mining.</strong>
+          </p>
+          <p>
+            <strong>4. Irrigation</strong> occurs in most areas of the country, but is larger in areas where rainfall is insufficient to meet crop what needs, such as in the <strong>drier parts of the West</strong>. Irrigation in <strong>eastern Arkansas</strong> provides water to flood rice fields as well as supplement rainfall to other crops.
+          </p>
+          <p>
+            <strong>5. Industrial withdrawals</strong> are driven by many factors. Industrial  facilities for production of steel in Lake County, Indiana, are located on Lake Michigan, a large source of fresh water, with access to water and land transportation, and ore and coal deposits. Lake County accounts for 8% of the US industrial water withdrawals.
+          </p>
+          <p>
+            <strong>6.</strong> The Gulf Coast is another hotspot of <strong>industrial</strong> withdrawals for the chemical and petroleum industries.
+          </p>
+          <p>
+            <strong>7. Thermoelectric</strong> power plants use steam to drive turbines and generate electricity. In the <strong>eastern U.S.</strong> where water is relatively abundant, large volumes of water often are withdrawn, used once for cooling, then returned to the source a little warmer than before. In the <strong>western U.S.</strong>, cooling water is more often withdrawn and recirculated many times, so less is withdrawn overall.
+  
+          </p>
+        </div>
+        
+        <p id="national-static-pie-mobile-interact" class="readMore"><strong>Expand to read more</strong></p>
```

Comment:
Again, I suspect that just "Read more" would work, though I haven't looked at the rendered version.

Reply:
The language doesn't really affect the functionality, so if you guys want less text it works for me.  For the CO River drought Viz we just had Read More and Hide swapping out and heard no complaints.